docker-rancher/server.json

```diff
@@ -1,348 +1,289 @@
-        "Basic_A1",
-        "Basic_A2",
-        "Basic_A3",
-        "Basic_A4",
-        "Standard_A5",
-        "Standard_A6",
-        "Standard_A7",
-        "Standard_D1",
-        "Standard_D2",
-        "Standard_D3",
-        "Standard_D4",
-        "Standard_D11",
-        "Standard_D12",
-        "Standard_D13",
-        "Standard_D14",
-        "Standard_D1_v2",
-        "Standard_D2_v2",
-        "Standard_D3_v2",
-        "Standard_D4_v2",
-        "Standard_D5_v2",
-        "Standard_D11_v2",
-        "Standard_D12_v2",
-        "Standard_D13_v2",
-        "Standard_D14_v2",
-        "Standard_DS1",
-        "Standard_DS2",
-        "Standard_DS3",
-        "Standard_DS4",
-        "Standard_DS11",
-        "Standard_DS12",
-        "Standard_DS13",
-        "Standard_DS14",
-        "Standard_G1",
-        "Standard_G2",
-        "Standard_G3",
-        "Standard_G4",
-        "Standard_G5",
-        "Standard_GS1",
-        "Standard_GS2",
-        "Standard_GS3",
-        "Standard_GS4",
-        "Standard_GS5",
-        "Standard_A8",
-        "Standard_A9",
-        "Standard_A10",
-        "Standard_A11"
-      ],
-      "defaultValue": "Standard_A2"
-    },
-    "rancherCount": {
-      "type": "int",
-      "metadata": {
-        "description": "Will be ignored, solely for integration purposes"
-      },
-      "minValue": 1,
-      "maxValue": 1
-    },
-    "location": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Location for all resources."
-      }
-    }
-  },
-  "variables": {
-    "rancherCount": 1,
-    "availabilitySet": "[parameters('rancherNodeName')]",
-    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'rancherhost')]",
-    "networkSecurityGroupName": "[parameters('rancherNodeName')]",
-    "imagePublisher": "Canonical",
-    "imageOffer": "UbuntuServer",
-    "ubuntuOSVersion": "14.04.5-LTS",
-    "virtualNetworkName": "[parameters('rancherNodeName')]",
-    "addressPrefix": "192.168.254.0/24",
-    "subnetName": "[parameters('rancherNodeName')]",
-    "subnetPrefix": "192.168.254.0/24",
-    "publicIPAddressType": "Dynamic",
-    "storageAccountType": "Standard_LRS",
-    "vmStorageAccountContainerName": "vhds",
-    "vmSize": "[parameters('rancherVmSize')]",
-    "extensionName": "DockerExtension",
-    "subnetRef": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnetName'))]"
-  },
-  "resources": [
-    {
-      "type": "Microsoft.Storage/storageAccounts",
-      "name": "[variables('storageAccountName')]",
-      "apiVersion": "2015-06-15",
-      "location": "[parameters('location')]",
-      "properties": {
-        "accountType": "[variables('storageAccountType')]"
-      }
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "apiProfile": "2019-03-01-hybrid",
```

Comment:
what's the intent for using this - you also have all the apiVersions specified so this isn't doing anything at the moment...